src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +14,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword().equals(password);
```

Comment:
think about how to rewrite it inline

Reply:
Sorry, I didn't get what should l do. What does mean inline?

Reply:
` if (user == null) {
            return false;
        }
        return user.getPassword().equals(password);`

it`s mean that you need write this code

in one line after return, because expression return you boolean value, so that you don`t need to write return true or return false

Reply:
if you have question ask it in chat

Reply:
But now check style fails because of this changes ![Screenshot_23](https://github.com/mate-academy/jv-oop-lesson-0/assets/12033542/b3b1e4ae-cbaa-44e3-bd27-f5c9787b7ff1)